src/main/java/com/aws/greengrass/logmanager/CloudWatchAttemptLogsProcessor.java

```diff
@@ -213,6 +215,8 @@ private boolean processLogLine(AtomicInteger totalBytesRead,
                                    long currentPosition,
                                    long lastModified) {
         boolean reachedMaxSize;
+        int expectedToAddSize = data.toString().getBytes(StandardCharsets.UTF_8).length;
```

Comment:
can we do this any cheaper? This is an expensive call.

Reply:
Computing only once now as it was done earlier